app/models/manageiq/providers/amazon/cloud_manager/event_target_parser.rb

```diff
@@ -0,0 +1,139 @@
+class ManageIQ::Providers::Amazon::CloudManager::EventTargetParser
```

Comment:
done, ManagerRefresh::Target and ManagerRefresh::TargetCollection are being introduced as a dependency